pkg/k8s/apis/cilium.io/client/crds/v2/ciliumnodes.yaml

```diff
@@ -0,0 +1,548 @@
+
```

Comment:
I think conflict resolution created a file here which does not belong to the v1.8 branch. Looking at the v1.8 branch, it seems the `examples/crds/` does not have a `ciliumnodes.yaml`, so it's safe to just remove this yaml file from the commit without replacement.

Reply:
Good catch, I didn't even notice the new file. I'm not very used to doing `am` process :'( 
I will remove the file

Reply:
> Good catch, I didn't even notice the new file. I'm not very used to doing am process :'(

For what it's worth, the core thing that we intend to do is make sure the backport looks the same as the original commit. So `git show` for both versions, and if there's any major differences, then ideally you either can explain why such a difference exists or you're able to pinpoint the missing commits underneath, or alternatively if neither of these options are reasonable, skip the backport and notify the original PR author that the backport is complex and the author should perform the backport to make sure it's correct for this branch.

Reply:
Maybe we still need to improve on the docs for this to make this clearer^.

Reply:
I think it's down to experience of doing backports. I had a different experience doing `am` rather than what I'm used to in `rebase` conflicts where the proposed diffs and conflicts were not properly show in editor until `git add .`. And at that point you it's easy to make a mistake like this and forget to check each file.

Perhaps on the tooling front we can automate the rough comparison of before/after patch sizes to make sure they are of comparable size 🤷